src/main/java/ch/njol/skript/expressions/ExprSpawnDelay.java

```diff
@@ -0,0 +1,138 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.Changer;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.block.Block;
+import org.bukkit.block.CreatureSpawner;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Spawner Spawn Delay")
+@Description({
+	"Gets or sets the spawn delay of a Spawner block. Can also be used to get the minimum or maximum value.",
+	"This value is used when the spawner resets its delay. Measured in ticks.",
+	"When setting minimum/maximum values, the spawner will choose a random number between the minimum and maximum delay to use as it's delay."
+})
+@Examples({
+	"set {_count} to minimum spawn delay of target block",
+	"set spawn delay of {_spawner} to 800"
+})
+@Since("INSERT VERSION")
+public class ExprSpawnDelay extends SimplePropertyExpression<Block, Integer> {
```

Comment:
Needs to work in terms of timespans, not integers.

Reply:
unresolved